ray-operator/controllers/ray/common/constant.go

```diff
@@ -9,6 +9,9 @@ const (
 	RayNodeLabelKey      = "ray.io/is-ray-node"
 	RayIDLabelKey        = "ray.io/identifier"
 
+	NameLabelKey      = "app.kubernetes.io/name"
```

Comment:
KubernetesApplicationNameLabelKey
KubernetesCreatedByLabelKey

same naming for `apiserver/pkg/util/config.go`